pygeoapi/models/provider/base.py

```diff
@@ -71,18 +71,33 @@ class TileMatrixSetEnumType(BaseModel):
     tileMatrixSet: str
     tileMatrixSetURI: str
     crs: str
+    tileMatrixSetDefinition: dict
 
 
 class TileMatrixSetEnum(Enum):
     WORLDCRS84QUAD = TileMatrixSetEnumType(
         tileMatrixSet="WorldCRS84Quad",
         tileMatrixSetURI="http://schemas.opengis.net/tms/1.0/json/examples/WorldCRS84Quad.json",  # noqa
```

Comment:
This is not a valid registered 2DTMS URI.

The correct URI is  http://www.opengis.net/def/tilematrixset/OGC/1.0/WorldCRS84Quad